### PR TITLE
fix the comments of default daemonset update strategy

### LINKED ISSUE #83007: update the daemonSet default updateStrategy type to OnDelete
**What happened**:

the default DaemonSetUpdateStrategyType was changed to Ondelete by default, but the document still say that the default value was `RollingUpdate`

refer doc https://kubernetes.io/docs/tasks/manage-daemon/update-daemon-set/#daemonset-update-strategy.

**What you expected to happen**:

**How to reproduce it (as minimally and precisely as possible)**:

**Anything else we need to know?**:

**Environment**:
- Kubernetes version (use `kubectl version`):
- Cloud provider or hardware configuration:
- OS (e.g: `cat /etc/os-release`):
- Kernel (e.g. `uname -a`):
- Install tools:
- Network plugin and version (if this is a network-related bug):
- Others:

### PR DESCRIPTION
**Pull request by danielqsj**
**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #83007

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

File: pkg/apis/apps/types.go [hunk 1]
```diff
@@ -502,8 +502,7 @@ type DeploymentList struct {
 
 // DaemonSetUpdateStrategy defines a strategy to update a daemon set.
 type DaemonSetUpdateStrategy struct {
-	// Type of daemon set update. Can be "RollingUpdate" or "OnDelete".
-	// Default is OnDelete.
+	// Type of daemon set update. Can be "RollingUpdate" or "OnDelete". Default is RollingUpdate.
```
Comment (liggitt) [inline pkg/apis/apps/types.go]: defaults are specific to external versions (e.g. apps/v1, etc). remove the info about the default from this internal type

Comment (danielqsj) [reply pkg/apis/apps/types.go]: sure, PR updated :)

**Comment by liggitt** [comment]
/lgtm
/approve

the pull-kubernetes-node-e2e job is failing right now. fix in https://github.com/kubernetes/test-infra/pull/14466

**Pull request merged**